data/campaigns/Descent_Into_Darkness/scenarios/10_Alone_at_Last.cfg

```diff
@@ -400,7 +454,7 @@
 
         [message]
             speaker=Malin Keshar
-            message= _ "Perhaps, but I have nothing left to lose."
+            message= _ "Perhaps... but I have nothing left to lose..."
```

Comment:
I'd skip the latter ... here. Ending with a period makes him sound like he actually realizes he really doesn't have anything left to lose, whereas trailing off gives more of an impression that he's just being emotional and lacks resolve (which he seems to have on the preceding line).

Reply:
The whole point is that he doesn't have resolve anymore. He doesn't know what else to do, so he just follows his gut instinct and finds himself a reason for trying to fight Darken Volk. Beyond that, he doesn't know what to do. (At least, this is my intention).